.github/workflows/cmake.yml

```diff
@@ -26,7 +22,7 @@ jobs:
       run: pip install -r third_party/requirements.txt
 
     - name: Configure CMake
-      run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} -DUR_ENABLE_TESTS=ON
+      run: cmake -B ${{github.workspace}}/build -DDEVELOPER_MODE=ON -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} -DBUILD_TESTS=ON
```

Comment:
Add DEVELOPER_MODE to the list of cmake options in the README (perhaps with info that it enables Werror compiler flag?)

Reply:
done